Log: add missing include for std::back_inserter()
e380ae90ebb6325d1820b6f34e10bf3474710899

Index: src/Log.cxx
--- src/Log.cxx.orig
+++ src/Log.cxx
@@ -23,6 +23,8 @@
 
 #include <fmt/format.h>
 
+#include <iterator> // for std::back_inserter()
+
 static constexpr Domain exception_domain("exception");
 
 void
